src/js/views/LottoView.js

```diff
@@ -1,7 +1,19 @@
-import { $, createElement } from '../utils.js';
+import { $, $all, createElement } from '../utils.js';
 
 export default class LottoView {
+  renderWinningResult(result) {
+    const { winningRankCounts, winningRate } = result;
```

Comment:
```suggestion
  renderWinningResult({ winningRankCounts, winningRate }) {
```

취향차이지만 코멘트 남겨봅니다!ㅋㅋㅋ

Reply:
ㅋㅋㅋㅋㅋ짚어주셔서 감사합니다!